types/test.tsx

```diff
@@ -57,14 +59,22 @@ async function testDebug() {
   debug(getAllByTestId('testid'))
 }
 
-async function testScreen() {
+export async function testScreen() {
   render(<button />)
 
-  screen.findByRole('button')
+  await screen.findByRole('button')
 }
 
-async function testWaitFor() {
+export async function testWaitFor() {
   const {container} = render(<button />)
   fireEvent.click(container)
   await waitFor(() => {})
 }
+
+/*
+eslint
+  testing-library/prefer-explicit-assert: "off",
+  testing-library/no-wait-for-empty-callback: "off",
+  testing-library/no-debug: "off",
+  testing-library/prefer-screen-queries: "off"
+*/
```

Comment:
Could we disable these globally or do they need to be enabled in other tests?

Reply:
We only need them in this test. So I don't see the point in doing that, but if we did create a separate file within this directory that needed these rules disabled then we could add these to an eslintrc file within this directory.